plugins/templates/removeTag.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+
+/* Currently, only build scope is allowed to tag template due to security reasons.
+ * The same pipeline that publishes the template has the permission to tag it.
+ */
+module.exports = () => ({
+    method: 'DELETE',
+    path: '/templates/tags',
+    config: {
+        description: 'Delete a template tag',
+        notes: 'Delete a specific template',
+        tags: ['api', 'templates'],
+        auth: {
+            strategies: ['token', 'session'],
+            scope: ['build']
+        },
+        plugins: {
+            'hapi-swagger': {
+                security: [{ token: [] }]
+            }
+        },
+        handler: (request, reply) => {
+            const pipelineFactory = request.server.app.pipelineFactory;
+            const templateFactory = request.server.app.templateFactory;
+            const templateTagFactory = request.server.app.templateTagFactory;
+            const pipelineId = request.auth.credentials.pipelineId;
+            const config = request.payload;
```

Comment:
The RESTful endpoint would be `DELETE /templates/{name}/tags/{tagName}`. You should be able to fetch the pipeline data from the template data and the JWT when comparing for authorization [build scope].

Alternatively, this could be `DELETE /templates/{name}/{tagName or version}` if it was less restful, but this adds a layer of confusion.